packs/deities/lao-shu-po.json

```diff
@@ -9,7 +9,7 @@
         ],
         "category": "deity",
         "description": {
-            "value": "<p><strong>Edicts</strong> Work quietly toward your goals in the shadows, steal what you need, keep an ear among the ignored and downtrodden</p>\n<p><strong>Anathema</strong> Work honestly for something you could steal instead, risk too much for another creature</p>"
+            "value": "<p><strong>Title</strong> Old Rat Woman</p>\n<p><strong>Areas of Concern</strong> Night, rats, thieves</p>\n<p><strong>Edicts</strong> Work quietly toward your goals in the shadows, steal what you need, keep an ear among the ignored and downtrodden</p>\n<p><strong>Anathema</strong> Work honestly for something you could steal instead, risk too much for another creature</p>\n<p><strong>Religious Symbol</strong> emaciated rat</p>\n<p><strong>Sacred Animal</strong> rat</p>\n<p><strong>Sacred Colors</strong> black, green</p>"
```

Comment:
Should the name of the deity be "Lao Shu Po" or "Lau Shu Po"? It's "Lao Shu Po" in the original LO:G&M, but "Lau Shu Po" in LO:DMws.

Reply:
Lao Shu Po is the correct name - the Web Supplement version is a typo